terminology_bot/install.sh

```diff
@@ -0,0 +1,9 @@
+docker kill pg_terminology
+docker rm /pg_terminology
+docker build -t eg_postgresql .
+docker run -p 5555:5432 -d --name pg_terminology eg_postgresql
+sleep 10
```

Comment:
After this line this script become unclear for me. Could you please clarify the purpose of the script?